libs/application/templates/parental-leave/src/forms/InReview.ts

```diff
@@ -5,12 +5,13 @@ import {
   Form,
   FormModes,
 } from '@island.is/application/core'
+
 import Logo from '../assets/Logo'
-import { mm } from '../lib/messages'
+import { inReviewForm as m, mm } from '../lib/messages'
```

Comment:
This seems fishy 🐟

Reply:
Yeah totally. The usage of the `import { m } from '.....'` is already used in every app so I didn't want to change the behavior in one app that would make the usage different.

A good alternative, in my opinion, would be to use the object directly and instead of using:

```typescript
formatMessage({
  id: 'm.my.key',
})
```

it would be:

```typescript
formatMessage({
  id: 'otherParentApprovalFormMessages.my.key',
})
```

more verbose, but clearer on what it is.